src/pip/_internal/basecommand.py

```diff
@@ -313,16 +313,24 @@ def populate_requirement_set(requirement_set, args, options, finder,
             if options.find_links:
                 raise CommandError(
                     'You must give at least one requirement to %(name)s '
-                    '(maybe you meant "pip %(name)s %(links)s"?)' %
-                    dict(opts, links=' '.join(options.find_links)))
+                    '(maybe you meant "pip %(name)s %(links)s"?)'
+                    % dict(opts, links=' '.join(options.find_links))
+                )
```

Comment:
I prefer `%` at the end of the line containing the format string - makes it more obvious that this is a formatting operation.

Reply:
I prefer not using the ``%`` operator at all ;)

PEP 8 basically says "we used to say break after, but people have successfully argued that break before is better, so either is OK, but it's recommended to break before", so black is following the *recommended* approach in PEP 8.

https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator